docs/samples/models/ibm-granite-2b-gpu/model.yaml

```
@@ -0,0 +1,91 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: LLMInferenceService
+metadata:
+  name: granite-2b-gpu
+  annotations:
+    alpha.maas.opendatahub.io/tiers: '[]'
+spec:
+  model:
+    uri: hf://ibm-granite/granite-3.1-2b-instruct
+    name: ibm-granite/granite-3.1-2b-instruct
+  replicas: 1
+  router:
+    route: { }
+    # Connect to MaaS-enabled gateway
+    gateway:
+      refs:
+        - name: maas-default-gateway
+          namespace: openshift-ingress
+  template:
+    containers:
+      - name: main
+        image: vllm/vllm-openai:v0.6.3.post1
+        command: ["python3", "-m", "vllm.entrypoints.openai.api_server"]
+        args:
+          - "--model=/mnt/models"
+          - "--served-model-name=granite-2b-instruct"
```

We usually do in other examples for consistency. Is `granite-2b-instruct` what we want to use here or the name we define above is fine too?

```suggestion
          - "--served-model-name \"{{ .Spec.Model.Name }}\""
```

---

By looking at our samples I wonder if we can follow the same approach and unify as follow-up?